tests/queries_test.go

```diff
@@ -1,64 +1,29 @@
 package tests
```

Comment:
I would add the a custom build tag here:
```suggestion
// +build system
package tests
```
Then change the `make test-system` command do this: `go test -tags=system ./test`. This will ensure this doesn't get run for `go test ./...`

Reply:
lovely